### ISSUE TITLE
Feature Request: XMPP+OMEMO 

### ISSUE THREAD

**Issue by JokerGermany** [TheGoddessInari/rambox#102]
It would be great if you could integrate jsxc
https://github.com/jsxc

**Comment by ryliejamesthomas** [comment]
I'd love to have the ability to use XMPP too!!

**Comment by alien2003** [comment]
There is also Converse.js. It would be great to have it supported by Rambox

**Comment by ryliejamesthomas** [comment]
Yeah, converse.js was what recommended over on the Rambox issue tracker: https://github.com/ramboxapp/community-edition/issues/38

**Comment by alien2003** [comment]
Can somebody post working script for Converse/Inverse here?

**Comment by alien2003** [comment]
@ryliejamesthomas can you please post a solution for Rambox-OS?

**Comment by ryliejamesthomas** [comment]
If I had one I'd share it. Don't @ me.

**Comment by alien2003** [comment]
Custom JS for login for Converse is:
```
document.getElementsByName("jid")[0].value = "username";
document.getElementsByName("password")[0].value = "password";
document.getElementsByClassName("btn")[0].click();
```
Custom update badge JS:
```
let checkUnread = () => {
    a=document.getElementsByClassName("msgs-indicator").length
    b=document.getElementsByClassName("unread-msgs").length
    hamsket.updateBadge(a,b)
};
setInterval(checkUnread, 3e3);
```

**Comment by TheGoddessInari** [comment]
You guys are wanting something like https://conversejs.org/fullscreen.html added as a service, or is there a different/better URL I'm missing?

Native support would be nice, but I'm not really looking to change the scope of the project until no longer depending on electron is an option (which would most likely entail a rewrite).

**Comment by alien2003** [comment]
Yes.
There is official instance here: https://inverse.chat/. It would be nice if you would add it as a service with the ability to set custom server address

**Comment by ryliejamesthomas** [comment]
Yep, the inverse.chat URL is the one I've been using.
